frontend/src/editor/types.ts

```diff
@@ -20,8 +20,8 @@ export type Document = {
 
 // we fire this event when the user clicks on a word and we want the player to skip through it
 export const TEXT_CLICK_EVENT = 'textClick';
-export class TextClickEvent extends CustomEvent<{ text: Text }> {
-  constructor(text: Text) {
-    super(TEXT_CLICK_EVENT, { detail: { text } });
+export class TextClickEvent extends CustomEvent<{ start?: number }> {
```

Comment:
If we do this, I would prefer to rename this event to something like `SeekEvent` or `SeekToEvent`